dashboard/app/models/sections/section.rb

```diff
@@ -81,7 +81,29 @@ def sti_name
   alias_attribute :lesson_extras, :stage_extras
 
   validates :participant_type, acceptance: {accept: Curriculum::SharedCourseConstants::PARTICIPANT_AUDIENCE.to_h.values, message: 'must be facilitator, teacher, or student'}
-  validates :grade, acceptance: {accept: [SharedConstants::STUDENT_GRADE_LEVELS, SharedConstants::PL_GRADE_VALUE].flatten, message: "must be one of the valid student grades. Expected one of: #{[SharedConstants::STUDENT_GRADE_LEVELS, SharedConstants::PL_GRADE_VALUE].flatten}. Got: \"%{value}\"."}
+
+  # Custom CSV serializer for the grades array.
+  class GradesArray
+    def self.dump(grades)
+      unless grades.is_a?(Array) || grades.nil?
+        raise ArgumentError, "Grades must be an array"
+      end
+      grades ? grades.uniq.join(',') : nil
```

Comment:
Should we sort grades here for consistency?

Reply:
oh nevermind, you do that in `load`!

Reply:
oh, I think this is a good point though, I want the value in the database to be in sorted order so RED can make use of it. that was part of the initial design spec. i'll put the sorting in `dump` instead.